code/modules/admin/verbs/shuttles.dm

```diff
@@ -0,0 +1,108 @@
+/proc/emergency_sanity_check()
+	if(SSshuttle.emergency.mode != SHUTTLE_IDLE)
+		var/confirm = alert(src, "Modification of the emergency shuttle while it is not idle can be highly dangerous, and may result in WEIRD UNPREDICTABLE SHIT. Are you SURE you want to continue? Obviously if you're not touching the emergency shuttle, then you're probably fine.", "Confirm", "Yes", "No")
+		if(confirm == "Yes")
+			return TRUE
+		else
+			return FALSE
+	else
+		return TRUE
+
+/client/proc/cmd_admin_destroy_shuttle()
+	set category = "Admin"
+	set name = "Shuttle Destroy"
+
+	if (!holder)
+		src << "Only administrators may use this command."
+		return
+
+	if(!emergency_sanity_check())
+		return
+
+	var/list/names = list()
+	var/obj/docking_port/mobile/M
+	for (var/atom/AM in SSshuttle.mobile)
+		M = AM
+		names += M.name
+
+	var/selected = input("Select shuttle to DESTROY", "Shuttles") in names
```

Comment:
I don't know if there's a way of adding cancel buttons to basic byond input(). It does ask afterwards. Obviously when this is all replaced by the shuttle manipulator tgui, then it won't matter.